monitoring/types.go

```diff
@@ -14,6 +14,7 @@ const (
 	subsystemDisk            = "disk"
 	subsystemInstanceManager = "instance_manager"
 	subsystemManager         = "manager"
+	subsystemBackup          = "backups"
```

Comment:
The change to value is also needed. backup not backups.

Reply:
Fixed in b92e5cc21e2211ccc124ef4234b3288a8d413e26